mirage/serializers/node.ts

```diff
@@ -47,6 +47,16 @@ export default class NodeSerializer extends ApplicationSerializer {
                     meta: this.buildRelatedLinkMeta(model, 'draftRegistrations'),
                 },
             },
+            affiliatedInstitutions: {
+                self: {
+                    href: `${apiUrl}/v2/nodes/${model.id}/relationships/institutions/`,
+                    meta: {},
+                },
```

Comment:
This is not currently used, but [EMB-227](https://openscience.atlassian.net/browse/EMB-227) is coming, so let's leave it in.